main/webapp/modules/core/scripts/views/data-table/data-table-view.js

```diff
@@ -99,11 +100,11 @@ DataTableView.prototype.render = function() {
       '<div class="viewpanel-sorting" bind="sortingControls"></div>' +
       '<div class="viewpanel-paging" bind="pagingControls"></div>' +
     '</div>' +
-    '<div bind="dataHeaderTableContainer" class="data-header-table-container">' +
-      '<table bind="headerTable" class="data-header-table"></table>' +
-    '</div>' +
     '<div bind="dataTableContainer" class="data-table-container">' +
-      '<table bind="table" class="data-table"></table>' +
+      '<table bind="table" class="data-table">'+
+        '<thead bind="headerTable" class="data-header-table">'+
+        '</thead>'+
+      '</table>' +
```

Comment:
Do we want to add an explicit `<tbody>` element here?

Reply:
The jquery function defined in render_table is generating the `<tbody>`, hence I left it. If needed I can revamp the container to have `<tbody>` explicitly.